src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/menu/MainMenu.kt

```diff
@@ -25,7 +32,30 @@ class MainMenu : MenuBar() {
                         }
                     }
                 }
+                menu("Default Audio Plugin") {
+                    graphic = MaterialIconView(MaterialIcon.MIC, "20px")
+                    pluginRepo
+                            .getAll()
+                            .observeOnFx()
+                            .subscribe { pluginData ->
```

Comment:
too much not-init logic going on here that probably needs to be in a usecase/viewmodel